docker-compose.dev.yaml

```
@@ -1,12 +1,22 @@
+      timeout: 5s
+      retries: 5
+    deploy:
+      resources:
+        limits:
+          memory: 1G
     ports:
-      - 27017:27017
+    - "27017:27017"
     volumes:
-      - mongodb-data:/data/db
+      - mongo_data:/data/db
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)